cursorless-talon/src/modifiers/glyph.py

```diff
@@ -0,0 +1,16 @@
+from talon import Module
+
+mod = Module()
+
+mod.list(
+    "cursorless_glyph_modifier",
+    desc="Cursorless glyph,  modifier",
+)
+
+
+@mod.capture(rule="{user.cursorless_glyph_modifier} <user.any_alphanumeric_key>")
+def cursorless_glyph_modifier(m) -> dict[str, str]:
+    return {
+        "type": "glyph",
```

Comment:
I think `"glyph"` is a nice spoken form, but I wonder whether it is the clearest identifier. I don't feel strongly one way or another. Could use something like `instanceOfChar`, `specificChar`. Or stick with `glyph`. @josharian any opinions?

Reply:
`glyph` seems fine to me. Why have multiple things to learn when you could have just one?

Reply:
One thing @AndreasArvidsson and I discussed is that this exact scope could be used for arbitrary text, not just single characters. Might be an argument for a different name. Might not. But could be something like `exactMatch`. 

Although the glyph scope should be case insensitive. But I guess there could be a flag on it eventually and then it could be for both

Reply:
literal? match? constantString? matchString?

Reply:
After a long discussion I reached consensus with myself and decided to call it `literal` :D

Reply:
Reverted to glyph